gadgets/trace_malloc/gadget.yaml

```diff
@@ -1,7 +1,7 @@
 name: trace malloc
 description: use uprobe to trace malloc and free in libc.so
 tracers:
-  events:
+  malloc:
```

Comment:
FYI, I picked up your commit in https://github.com/inspektor-gadget/inspektor-gadget/pull/2636 along other changes.

See also @eiffel-fl's request in https://github.com/inspektor-gadget/inspektor-gadget/pull/2586#discussion_r1533435119

Reply:
Thanks, dropped that commit in this PR.